src/test/resources/features/AndreiCiuvaga.feature

```diff
@@ -0,0 +1,40 @@
+@andreiciuvaga
+Feature:Andrei's Feature
```

Comment:
rename feature and feature files related to business

Reply:
Ok, thanks